### DISCUSSION TITLE
Why `tailwind` docs was removed from the website?

### DISCUSSION THREAD

**Discussion by pawojciechowski** [cschroeter/park-ui#367] [Q&A] [answered]
I see there is no `tailwind` integration option on the website https://park-ui.com/react/docs/overview/getting-started. What are the plans for supporting tailwind?

**Comment by cschroeter** [comment, marked as answer]
With a heavy heart, I have decided to discontinue support for Tailwind as the initial maintainer no longer has time to maintain it.
In addition the project will move to the chakra-ui org.

**Reply by pawojciechowski** [reply]
Sad to hear that 😿 This project had awesome connection examples of 2 completely different styling libraries with the ArkUI. This was inspiring to me! I mean I love ChakraUI (and use it extensively). I think PandaCSS is an awesome library too, but having Panda & Tailwind implemented in a way you can switch between them with a button was one of the best things I have seen in the UI libraries world. 

Sometimes there are restrictions on the styling library you may choose for a project, and it would be awesome to have an example of using Tailwind library with ArkUI.

**Reply by tclain** [reply]
@cschroeter would it be possible to let the community take over then  / in a different branch ? I'd be happy to help out

 We 've been investing massively into the tailwind version in court startup, coming from chakra v2 (I love panda, but the tailwind ecosystem is really strong and in many ways park UI seems to me what chakra 3 should have been).
As other mentioned, it being cross-framework also allows building SDK on top of it a dream.

We have a vested interest in my company to make it live. it's a shame only shadcn UI exists as a comp.

**Comment by btroop** [comment]
i've been building my entire project on park-ui components w/ tailwind...if I understand this correctly, I'm going to have to scrap the entire thing and either a) redo it with panda or b) move to a different library, like shad-cn. is that correct???

**Reply by jackvane** [reply]
Same here, now I'm expected to refactor all of my components and learn a new styling system in order to continue using this library. If this was React, I'd just drop the project entirely and use something predictable like shadcn/ui, but Solid projects are pretty much dead in the water now because the alternatives are still that much worse.

Removing support for what is arguably the most popular CSS library without a migration guide is pretty much the death knell to a project like this in React. There's also no reason now to use this over just using Ark UI directly now considering the benefit was to not have to configure default styles with the library of our choice, which we now have to do **all of a sudden**.

**Reply by Spooky-0** [reply]
same ...
But to be honest, all you would have to do is to replace the variants, no?
e.g.
```
  md: {
        root: {
          gap: '3',
        },
        control: {
          width: '9',
          p: '0.5',
        },
        thumb: {
          width: '4',
          height: '4',
        },
        label: {
          textStyle: 'md',
        },
      },
```
with something like
```
  md: {
        root: "gap-4",
        control: "w-9 h-0.5",
        thumb: "w-4 h-4",
        label: "text-md"
      }
```

**Reply by aguilera51284** [reply]
It is certainly a shame that they no longer support tailwindcss, I have a startup that uses this with solid, now I will migrate to another component system. And to tell the truth I agree that at least for react, it is not worth using this compared to other ui

**Reply by baueer** [reply]
We've chose this library for the fresh looks, kinda different for the others out here in the React ecosystem. However, discontinuing support for one of the most widely used CSS libraries without a proper announcement and failing to ensure backward compatibility is a deal breaker for our project. Consequently, we are likely to switch to another component library in the future.

**Reply by tclain** [reply]
@baueer , I am planning to fork it, would you be interested ?

**Reply by gnosticdev** [reply]
You may have noticed that the tailwind version never was a "true" tailwind version. The parkwind plugin was adding the tailwind support via codegen, plus extending the tailwind config theme with `fg, bg, accent,...` colors. 

The actual classes used on the components that you see, are things like `dialog__trigger`, or `select__root` are just regular css classes. I actually remade many of the components with pure tailwind bc of this.

It can be done if someone wanted to take it on, I could contribute some tailwind versions of the components.

**Reply by tyteen4a03** [reply]
I use both Tailwind and PandaCSS in various projects, and I would encourage skeptics to give PandaCSS a fair trial. Chakra v3 would not have made sense with Tailwind syntax anyway.
